scripts/apps/search/components/fields/translations.tsx

```diff
@@ -70,7 +70,7 @@ export class Translations extends React.PureComponent<IProps> {
                     >
                         {'('}<b>{this.props.item.translations.length}</b>{')'}
                         {' '}
-                        {gettextPlural(this.props.item.translations.length, 'translation', 'translations')}
+                        {this.props.item.translations.length === 1 ? gettext('Translation') : gettext('Translations')}
```

Comment:
you should keep `gettextPlural` - other languages have more plural forms than one that depend on item count (in this case `translations.length`) and when the script is doing string extraction, it will mark it as plural. As a result translators would have to set 3-4 or how many plural forms there are in their language.

Reply:
yep problem is that it doesn't work atm, those parts don't show up on transifex

Reply:
what about .pot file?

Reply:
they are in the `.pot`, that part seems ok. will investigate it further this is rather temporary fix